src/components/DayPicker.jsx

```diff
@@ -288,22 +290,36 @@ export default class DayPicker extends React.Component {
   }
 
   renderNavigation() {
+    const {
+      navPrev,
+      navNext,
+    } = this.props;
+
     const isVertical = this.isVertical();
 
+    let navPrevIcon = navPrev;
+    let navNextIcon = navNext;
+    if (!navPrevIcon) {
+      navPrevIcon = isVertical ? <ChevronUp /> : <LeftArrow />;
```

Comment:
I thought about that initially, but my main argument in my head against it was. "isVertical is a prop being passed in like navPrev/navNext". E.g. they already know which is being rendered because the caller themselves is passing in the property.

That being said if there's a situation I'm not thinking of where they are unsure of it being vertical/not vertical, or we just determine it's "better" :tm: I'd be happy to make that change.